store/rootmulti/store.go

```diff
@@ -1035,8 +1035,14 @@ func commitStores(version int64, storeMap map[types.StoreKey]types.CommitKVStore
 	storeInfos := make([]types.StoreInfo, 0, len(storeMap))
 
 	for key, store := range storeMap {
-		commitID := store.Commit()
-
+		last := store.LastCommitID()
+		var commitID types.CommitID
```

Comment:
```suggestion

		var commitID types.CommitID
```

nit:  Can we please add a comment in this block as to what and why we're doing this pls?

Reply:
done